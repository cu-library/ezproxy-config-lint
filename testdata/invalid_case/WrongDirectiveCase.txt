title Siku Quanshu
URL http://skqs.yourlib.org
DJ skqs.yourlib.org
