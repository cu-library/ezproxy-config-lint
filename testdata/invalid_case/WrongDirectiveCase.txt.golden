testdata/invalid_case/WrongDirectiveCase.txt:1: title Siku Quanshu ‚Üê "title" directive does not have the right letter casing. It should be replaced by "Title" (L5001)
